lib/platform/github/index.ts

```diff
@@ -746,7 +747,12 @@ export async function getPr(prNo: number): Promise<Pr | null> {
   );
   const pr = (
     await githubApi.getJson<any>(
-      `repos/${config.parentRepo || config.repository}/pulls/${prNo}`
+      `repos/${config.parentRepo || config.repository}/pulls/${prNo}`,
+      {
+        headers: {
+          accept: 'application/vnd.github.merge-info-preview+json',
```

Comment:
I think it is applicable to GraphQL only, so is not required here

Reply:
Yes, removed it